packages/component/src/LinkDefinition/private/ItemBody.tsx

```diff
@@ -4,11 +4,20 @@ import Badge from './Badge';
 import OpenInNewWindowIcon from './OpenInNewWindowIcon';
 
 type Props = Readonly<{
-  badgeName?: string;
-  badgeTitle?: string;
+  // The text (usually a number) displayed at the head of the citation
   identifier?: string;
-  isExternal?: boolean;
+
+  // The text displayed as the main link of the citation
   text: string;
+
+  // do we show the external link icon after the link?
```

Comment:
Should probably answer this before merging.

Reply:
That's a description of the boolean field. I'll make that clearer.